packages/dds/tree/package.json

```diff
@@ -0,0 +1,96 @@
+{
+  "name": "@fluidframework/tree",
+  "version": "0.56.0",
+  "private": true,
+  "description": "Tree",
+  "homepage": "https://fluidframework.com",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/FluidFramework.git",
+    "directory": "packages/dds/tree"
+  },
+  "license": "MIT",
+  "author": "Microsoft and contributors",
+  "sideEffects": false,
+  "main": "dist/index.js",
+  "module": "lib/index.js",
+  "types": "dist/index.d.ts",
+  "scripts": {
+    "build": "concurrently npm:build:compile npm:lint && npm run build:docs",
+    "build:commonjs": "npm run tsc && npm run build:test",
+    "build:compile": "concurrently npm:build:commonjs npm:build:esnext",
+    "build:docs": "api-extractor run --local --typescript-compiler-folder ../../../node_modules/typescript && copyfiles -u 1 ./_api-extractor-temp/doc-models/* ../../../_api-extractor-temp/",
+    "build:esnext": "tsc --project ./tsconfig.esnext.json",
+    "build:full": "npm run build",
+    "build:full:compile": "npm run build:compile",
+    "build:test": "tsc --project ./src/test/tsconfig.json",
+    "ci:build:docs": "api-extractor run --typescript-compiler-folder ../../../node_modules/typescript && copyfiles -u 1 ./_api-extractor-temp/* ../../../_api-extractor-temp/",
+    "clean": "rimraf dist lib *.tsbuildinfo *.build.log",
+    "eslint": "eslint --format stylish src",
+    "eslint:fix": "eslint --format stylish src --fix --fix-type problem,suggestion,layout",
+    "lint": "npm run eslint",
+    "lint:fix": "npm run eslint:fix",
+    "test": "npm run test:mocha",
+    "test:coverage": "nyc npm test -- --reporter xunit --reporter-option output=nyc/junit-report.xml",
+    "test:mocha": "mocha --recursive dist/test --exit -r node_modules/@fluidframework/mocha-test-setup -r source-map-support/register --unhandled-rejections=strict",
+    "test:mocha:verbose": "cross-env FLUID_TEST_VERBOSE=1 npm run test:mocha",
+    "tsc": "tsc",
+    "tsfmt": "tsfmt --verify",
+    "tsfmt:fix": "tsfmt --replace"
+  },
+  "nyc": {
+    "all": true,
+    "cache-dir": "nyc/.cache",
+    "exclude": [
+      "src/test/**/*.ts",
+      "dist/test/**/*.js"
+    ],
+    "exclude-after-remap": false,
+    "include": [
+      "src/**/*.ts",
+      "dist/**/*.js"
+    ],
+    "report-dir": "nyc/report",
+    "reporter": [
+      "cobertura",
+      "html",
+      "text"
+    ],
+    "temp-directory": "nyc/.nyc_output"
+  },
+  "dependencies": {
+    "@fluidframework/common-utils": "^0.32.1",
+    "@ungap/structured-clone": "^0.3.4"
+  },
+  "devDependencies": {
+    "@fluidframework/build-common": "^0.23.0",
+    "@fluidframework/eslint-config-fluid": "^0.25.0",
```

Comment:
all these @fluidframework versions look incorrect, you need to align with the other packages, as they are all in the same lerna mono repo

Reply:
@yann-achard-MS - Package version mismatches are probably the root cause of the mocha-test-setup failure.  It's tedious, but you'll want to pull the latest from main and copy the versions currently in use by other nearby DDS packages.

Reply:
It was, indeed. Thank you for pointing that out.

@anthony-murphy - You specifically mentioned that *all* versions were incorrect but `build-common` and `common-utils` seems to have been correct as far as I can tell from looking at other packages. I'm guessing you just meant that they all needed to be checked for correctness, but I want to double-check with you in case there's still something I'm missing.

Reply:
i was just overzealous in my use of the word all. I didn't know the correct versions offhand and was just trying to convey they all need to be aligned